modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/CassandraRepositoryModule.java

```diff
@@ -152,11 +162,27 @@ protected void configure() {
         bind(ExperimentAuditLogAccessor.class).toProvider(ExperimentAuditLogAccessorProvider.class).in(Singleton.class);
         //Bind those count
         bind(BucketAssignmentCountAccessor.class).toProvider(BucketAssignmentCountAccessorProvider.class).in(Singleton.class);
+        bind(HourlyBucketCountAccessor.class).toProvider(HourlyBucketCountAccessorProvider.class).in(Singleton.class);
         //Bind those export
         bind(UserAssignmentExportAccessor.class).toProvider(UserAssignmentExportAccessorProvider.class).in(Singleton.class);
         //Bind assignments Count thread pool executor
         bind(ThreadPoolExecutor.class).annotatedWith(named("AssignmentsCountThreadPoolExecutor")).to(AssignmentCountExecutor.class).in(Singleton.class);
 
+        //Bind Scheduled Executor Service
+        bind(ScheduledExecutorService.class).annotatedWith(named(ASSIGNMENTS_HOURLY_AGGREGATOR_SERVICE)).to(AssignmentCountScheduledExecutorService.class).in(Singleton.class);
+        //This is the assignment aggregation interval.
+        Integer assignmentAggregationIntervalInMinutes = 2;
```

Comment:
This will be changed to 60 minutes for hourly aggregation, but it is 2 at the moment for testing purposes.